content/en/docs/Tasks/update-and-ship-operator.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Update and ship an Operator"
+date: 2021-03-10
+weight: 2
+description: > 
+    In depth explanation of operator update and shipment process. 
+--- 
+
+## Introduction
+
+In the Operator Lifecycle Manager (OLM) ecosystem, the following resources are used to resolve Operator installations and upgrades:
+
+1. `ClusterServiceVersion (CSV)` - A YAML manifest created from Operator metadata that assists the Operator Lifecycle Manager (OLM) in running the Operator in a cluster.
+
+    A CSV is the metadata that accompanies an Operator container image, used to populate user interfaces with information like its logo, description, and version. It is also a source of technical information needed to run the Operator, like the RBAC rules it requires and which Custom Resources (CRs) it manages or depends on.
+
+    A CSV is composed of a Metadata, Install strategy, and CRDs.
+
+2. `CatalogSource` - Operator metadata, defined in CSVs, can be stored in a collection called a CatalogSource. CatalogSource contains metadata that OLM can query to discover and install operators with their dependencies.
```

Comment:
I am not sure if these instructions are accurate. I think we need to explain how to the things with the Bundle Format.  See an example: https://raw.githubusercontent.com/operator-framework/olm-docs/master/content/en/docs/best-practices/common.md